client/dogs/FormPage.js

```diff
@@ -652,6 +671,69 @@ export default class IndexPage extends React.Component {
             </div>
 
             <div className="txt">
+              <label htmlFor="lookup-rabiesCert" className="txt-l">
+                Rabies Certificate
+              </label>
+              <Dropzone
+                disableClick={false}
+                multiple={false}
+                accept={'image/*'}
+                onDrop={this.onDropRabies.bind(this)}
```

Comment:
Because you've defined the `onDropRabies` and `onDropSpaying` methods with `method = () => {}` above, they're already bound to `this` so you don't need the `.bind` again. (It's good practice not to call `bind` in a `render` method to avoid creating unnecessary closures. Doing it once or twice won't be a performance problem, but if it's done once for every item in a big list it adds up.)